samples/net/sockets/echo/CMakeLists.txt

```diff
@@ -1,19 +1,9 @@
 cmake_minimum_required(VERSION 3.13.1)
-macro(set_conf_file)
-  if(EXISTS ${APPLICATION_SOURCE_DIR}/prj_${BOARD}.conf)
-    set(CONF_FILE "${APPLICATION_SOURCE_DIR}/prj_${BOARD}.conf")
-  elseif(EXISTS ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf)
-    set(CONF_FILE
-      "prj.conf ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf")
-  else()
-    set(CONF_FILE "prj.conf")
-  endif()
-endmacro()
 
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(sockets_echo)
 
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})
 
-include($ENV{ZEPHYR_BASE}/samples/net/common/common.cmake)
\ No newline at end of file
+include($ENV{ZEPHYR_BASE}/samples/net/common/common.cmake)
```

Comment:
Unrelated change

Reply:
True, but, we should have a newline at the end of the file.

I can create another commit for this, but I would prefer to leave it as-is.

Reply:
Adding a new commit into this set should not be a big issue. Don't you need to remove the extra CONFIG_I2C_VIRTUAL settings anyway?